src/postgres/src/backend/access/yb_access/yb_scan.c

```diff
@@ -2316,7 +2316,7 @@ YbBindHashKeys(YbScanDesc ybScan)
 		Assert(YbIsHashCodeSearch(key));
 		YbBound bound = {
 			.type = YB_YQL_BOUND_VALID,
-			.value = key->sk_argument
+			.value = (lc->data.int_value<0)?0:key->sk_argument
```

Comment:
Please leave a space around all the operators. Please look around the codebase for examples of how `ListCell` instances are accessed.

Can you achieve the same thing by only checking something on `key->sk_argument`?